lib/logflare/source/rate_counter_server.ex

```diff
@@ -268,14 +268,28 @@ defmodule Logflare.Source.RateCounterServer do
     Sources.Counters.get_inserts(source_id)
   end
 
+  @doc """
+  Takes a list of integers representing per second counts of events
+  of a source and returns the `avg` and `sum` of those counts.
+
+  ## Examples
+
+      iex> Logflare.Source.RateCounterServer.stats([12,5,30,5,1000])
+      %{avg: 211, sum: 1052}
+
+  """
+
+  @spec stats([integer()]) :: %{avg: integer(), sum: integer()}
   def stats(xs) when is_list(xs) do
     {total, count} =
       Enum.reduce(xs, {0, 0}, fn v, {total, count} ->
         {total + v, count + 1}
       end)
 
+    avg = Kernel.ceil(total / count)
```

Comment:
Ceil there is correct function? I would need to run benchmark whether this or `div(total + count - 1, count)` is faster.

Reply:
I think it is for rounding

Reply:
Ya it’s for rounding. You can take this over to get it in the latest version.

Reply:
Long floats were on the dashboard.

Reply:
@Ziinc I know that this is for rounding. Just my point is whether `trunc/1` (faster) or `round/1` (more correct) would not work better.